regression/contracts/nonvacuous_loop_contracts/main.c

```diff
@@ -0,0 +1,36 @@
+#include <assert.h>
+#include <stdlib.h>
+
+#define MAX_SIZE 1000000
+
+int main()
+{
+  size_t size;
+  __CPROVER_assume(size <= MAX_SIZE);
```

Comment:
nitpick: would it be nice a comment here.

Reply:
I realized that we don't need this `MAX_SIZE` at all, removing it.